ehr2meds/preMEDS/normalizer.py

```
@@ -162,30 +140,30 @@ def get_lab_values(self):
             value = pd.to_numeric(value)
 
-        if self.normalisation_type == "Min_max":
-            return self.min_max_normalise(concept, value)
-        elif self.normalisation_type == "Quantiles":
+        if self.normalization_type == "Min_max":
+            return self.min_max_normalize(concept, value)
+        elif self.normalization_type == "Quantiles":
             return self.quantile(concept, value)
         else:
-            Warning(f"Normalisation type {self.normalisation_type} not implemented")
+            warnings.warn(f"Normalization type {self.normalization_type} not implemented")
```

Ruff (0.15.10)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)